.circleci/scripts/update_config.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import os
+import re
+import subprocess
+import yaml
+
+
+def indent(text, indent=4):
+    """indent a text with n spaces"""
+    return '\n'.join('{}{}'.format(' ' * indent, line) for line in text.split('\n'))
```

Comment:
Do you know the `textwrap` module [1]? I discovered this recently.

[1] https://docs.python.org/3/library/textwrap.html#textwrap.indent

Reply:
I did not! Thx for the tip!